docs/testbed/README.testbed.SnappiTests.md

```diff
@@ -0,0 +1,344 @@
+# Running Snappi Tests and Snappi Configuration
+
+### Author
+
+| Version | Name          | Role          |
+| ------- | ------------- | ------------- |
+| 1.0     | Dev Ojha      | Spec Guide    |
+
+
+## 1 Physical Topology
+
+### 1.1 Purpose
+
+Each DUT front-panel port is cabled 1-to-1 to a traffic-generator port of equal speed so that Snappi test scripts can exercise link-level features and inject/measure traffic end-to-end.
+
+### 1.2 Sample link map
+
+| StartDevice | StartPort | EndDevice | EndPort | Bandwidth (Mb/s) | VLAN ID | VLAN Mode |
+| ----------- | --------- | --------- | ------- | ---------------- | ------- | --------- |
+| `dut1`      | Ethernet1 | `tgen1`   | P1      | 100 000          |         | Access    |
+| `dut1`      | Ethernet2 | `tgen1`   | P2      | 100 000          |         | Access    |
+
```

Comment:
1. Is there any requirement on the topology?
2. Since there are uplink and downlink ports, do we care which port to be connected to traffic generator?

Reply:
Do we care about the topo that used in snappi test? It seems that the design is to use `tgen` topo, but the example is showing `t0` topo.